### ISSUE TITLE
Run GUI Program on WSL1

### ISSUE THREAD

**Issue by bk3810** [bk3810/LinuxTips#2]
실행시 에러남.

export DISPLAY=localhost:0 확인...이미 저장되어 있을 수 있음.

sudo apt install dbus-x11

이후 실행은 되지만, GUI 프로그램 실행할때마다  다음 에러 출력됨.
# Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-wzn00x6fxh: Connection refused
=>

Adding the line

NO_AT_BRIDGE=1

to /etc/environment (or export NO_AT_BRIDGE=1 on console) solved the problem for me.
